dispatcher_plugin_nb2workflow/products.py

```diff
@@ -174,6 +174,27 @@ def write(self):
     def get_html_draw(self):
         return {'image': {'div': '<br>'+self.data_prod, 'script': ''} }
 
+
+class NB2WProgressProduct(NB2WProduct):
+    # TODO to be adapted
+    type_key = 'http://odahub.io/ontology#ODAProgressProduct'
```

Comment:
`type_key` is not really needed as far as the instance is directly created, not with the factory

Reply:
and in which cases is it needed?
should it be just removed for this particular class ?

Reply:
I don't see any reason to add such a term to the ontology, therefore there is no reason to keep it in the code.

If you just remove it, I think https://github.com/oda-hub/dispatcher-plugin-nb2workflow/blob/908cda39605dc3f62f3fc9773374b1c13cda47de/dispatcher_plugin_nb2workflow/products.py#L76 will break, but it can be easily adapted.

"Ordinary" products are all constructed according to their URIs by the factory function, which is called here https://github.com/oda-hub/dispatcher-plugin-nb2workflow/blob/908cda39605dc3f62f3fc9773374b1c13cda47de/dispatcher_plugin_nb2workflow/queries.py#L138, but this particular product is only initiated in the else clause

Reply:
true, `prod_list_factory` is not called for `NB2WProgressProduct`, so, at the moment, removal of `type_key` won't break